### ISSUE TITLE
NullReferenceException when sharing F# value with JavaScript

### ISSUE THREAD

**Issue by cartermp** [dotnet/interactive#532]
Using version 1.0.131001+ff9b940c966abf0fc4c341f432c45dc2e116b42b

Sharing an F# value (just a string) with JavaScript leads to an NRE:

![image](https://user-images.githubusercontent.com/6309070/84420731-502e1a00-abcf-11ea-81ce-42352007dfa2.png)

The `.dib` file:

```
#!fsharp

let myvalue = "Phillip"

#!html

<b id="target">HTML!</b>

#!javascript

let e = document.getElementById('target');
interactive.fsharp.getVariable('myvalue').then(data => e.uinnerText = data);
```

**Comment by cartermp** [comment]
This no longer reproduces.

```
#!fsharp

let myvalue = "Phillip"

#!html

<b id="target">HTML!</b>

#!javascript

let e = document.getElementById('target');
interactive.fsharp.getVariable('myvalue').then(data => e.innerText = data);
```

yields

![image](https://user-images.githubusercontent.com/6309070/89350216-766ab600-d664-11ea-9085-00d642fa4beb.png)

However, I am still struggling (perhaps due to my own personal problems or something related to this: https://github.com/dotnet/interactive/issues/531)

**Comment by colombod** [comment]
this was probably caused by the same bug reported in #657 and addressed with pr #682
